src/conf_mode/interfaces-openvpn.py

```diff
@@ -274,7 +274,7 @@ def verify(openvpn):
             elif v6remAddr and not v6loAddr:
                 raise ConfigError('IPv6 "remote-address" requires IPv6 "local-address"')
 
-            if (v4loAddr == v4remAddr) or (v6remAddr == v4remAddr):
+            if (v4loAddr == v4remAddr and len(v4loAddr) != 0) or (v6remAddr == v4remAddr):
```

Comment:
I think there is a bug in the original implementation. Shouldn't this be:

`if (v4loAddr == v4remAddr) or (v6loAddr == v6remAddr):`?

I'd rather implement a `vyos.util.is_list_equal()` helper which checks if two lists have `len() > 0` and the content is equal (unordered) - see https://thispointer.com/python-check-if-two-lists-are-equal-or-not-covers-both-ordered-unordered-lists/ or write:

`if (v4loAddr and v4remAddr and v4loAddr == v4remAddr) or (v6loAddr  and v6remAddr and v6loAddr == v6remAddr):`

Reply:
Done!